custom_connectors/oauth2/bim_360.rb

```diff
@@ -3324,22 +2289,27 @@
                        'filter[type]=items&filter[lastModifiedTimeRollup]-ge=' \
                        "#{last_modified_time}&" \
                        "page[limit]=#{limit}&page[skip]=#{skip}"
-                     get("/data/v1/projects/#{project_id}/folders/" \
-                       "#{folder_id}/contents", query_params)
+                     get("/data/v1/projects/#{input['project_id']}/folders/" \
+                       "#{input['folder_id']}/contents", query_params)
```

Comment:
you can use `folder_id ` field instead of `input['folder_id']`

Reply:
same comment as above. this ensures we're looking at the folder specified in the input instead of the cached value.